src/axislike.rs

```diff
@@ -30,6 +30,8 @@ pub struct SingleAxis {
     pub negative_low: f32,
     /// Whether to invert output values from this axis.
     pub inverted: bool,
```

Comment:
We can replace `inverted` with a negative sensitivity value.

Reply:
We could do this mathematically, but I think it makes the API a lot less clear. I think we should keep them separate, and add some kind of non-negative constraint on sensitivity (or at least a warning).

Reply:
Okay, fair. Let's do that then.

Reply:
Sorry, but what does "that" refer to? I'm guessing merging sensitivity and inverted as you said below, but what do about making that making the API less clear as plof27 said. I'm guessing just write useful docs on sensitivity to explain how inverting works now?

Reply:
As a side note, instead of removing the invert functions, I can just invert the sensitivity to do the same thing.
```rust
/// Returns this [`SingleAxis`] inverted.
#[must_use]
pub fn inverted(mut self) -> Self {
    self.sensitivity = -self.sensitivity;
    self
}
```

Reply:
> Ah, I like that :)
So, would you like that re-merged into one value? Or, close this as resolved and keep them separate. I am good with either solution.